doc/sphinx/changes.rst

```diff
@@ -1661,14 +1661,14 @@ Changes in 8.11+beta1
   (convenience) step was only performed when `R1` was Leibniz' `eq` or
   `iff`. Now, it is also performed for any relation `R1` which has a
   ``RewriteRelation`` instance (a `RelationClasses.Reflexive` instance
-  being also needed so :tacn:`over` can discharge the ``'Under[ _ ]``
+  being also needed so `over` can discharge the ``'Under[ _ ]``
```

Comment:
It's hard to comment on stuff that comes from another PR, but is the removal of these `:tacn:` really necessary?

Reply:
Pretty much.  There is no `over` tactic defined in the mlgs nor as a `Tactic Notation`.  The SSR chapter still has `.. tacn:: over ...`, so Sphinx doesn't complain.

I pulled in some of the SSR changes to make the merge easier later on.

Reply:
Still, I don't get how the changes to this particular file are related to the rest of the PR. So if they are not, it would be preferable to keep them for the other one.

Reply:
OK